BrowserKit/Sources/ToolbarKit/LocationView.swift

```diff
@@ -39,11 +40,30 @@ class LocationView: UIView, UITextFieldDelegate, ThemeApplicable {
     private lazy var gradientLayer = CAGradientLayer()
     private lazy var gradientView: UIView = .build()
 
+    private var clearButtonWidthConstraint: NSLayoutConstraint?
+    private var gradientViewWidthConstraint: NSLayoutConstraint?
+
+    private lazy var clearButton: UIButton = .build { button in
+        button.setImage(
+            UIImage(named: StandardImageIdentifiers.Large.crossCircleFill)?.withRenderingMode(.alwaysTemplate),
+            for: .normal
+        )
+        button.addTarget(self, action: #selector(self.clearURLText), for: .touchUpInside)
+        button.accessibilityLabel = "Clean"
```

Comment:
As this is in ToolbarKit we shall not use hard coded strings or a11y ids. These should be passed in through the `configure` method.